tool/ci/setup.sh

```diff
@@ -22,7 +21,6 @@ function flutter {
 }
 export -f flutter
 
-# TODO: Also support windows on github actions.
 if [[ $RUNNER_OS == "Windows" ]]; then
```

Comment:
is this if block necessary? It doesn't look like we manually install google chrome for the ubuntu OS. Looking at the comment, this is linking to an article from travis-ci, which we haven't used in 4 or 5 years.

Reply:
Good point - according to https://github.com/actions/runner-images/blob/main/images/windows/Windows2022-Readme.md#browsers-and-drivers it's already installed, so this shouldn't be required. I've removed it - if the bots turn red I'll investigate.